payjoin/src/core/send/v2/mod.rs

```diff
@@ -677,18 +623,21 @@ mod test {
         // ensure that the other builder methods also enable output substitution
         let req_ctx = SenderBuilder::new(PARSED_ORIGINAL_PSBT.clone(), pj_uri.clone())
             .build_non_incentivizing(FeeRate::BROADCAST_MIN)
+            .expect("build on test vectorshould succeed")
```

Comment:
Typo. Odd that spellcheck didnt pick this up 
```suggestion
            .expect("build on test vector should succeed")
```